plugins/chat/spec/db/migrate/20221201024458_make_channel_slugs_unique_with_index_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+#
+require Rails.root.join("plugins/chat/db/migrate/20221201024458_make_channel_slugs_unique_with_index.rb")
+
+RSpec.describe MakeChannelSlugsUniqueWithIndex do
```

Comment:
Will delete this test after review

Reply:
Thanks for adding this. Makes reviewing much better ❤️

Reply:
No problem! I need to for my sanity with these kind of changes hah